cyclops/evaluate/evaluator.py

```diff
@@ -141,13 +147,20 @@ def evaluate(  # pylint: disable=too-many-function-args
     if models is not None:
         if feature_columns is None:
             raise ValueError(
-                "Got `model` but `feature_columns` is None. Please specify "
-                "`feature_columns`."
+                "Got `models` but `feature_columns` is None. Please specify "
+                "`feature_columns` argument."
             )
         models = _prepare_models(models)
-        # TODO: compute predictions for each model and add them to the dataset
-        # with name `prediction_column_prefix.model_name`
-        raise NotImplementedError
+        for model_name, model in models.items():
```

Comment:
Is there a way to discard those predictions that are not present in the dataset for evaluation? Like those pathologies that are not included in the NIH dataset.

Reply:
No. Not at the moment.

Reply:
We can come back to this later then.